src/logic/currencyValues/store/actions/fetchCurrencySelectedValue.js

```diff
@@ -7,15 +7,16 @@ import { AVAILABLE_CURRENCIES } from '~/logic/currencyValues/store/model/currenc
 import type { GlobalState } from '~/store'
 
 // eslint-disable-next-line max-len
-const fetchCurrencySelectedValue = (currencyValueSelected: $Keys<typeof AVAILABLE_CURRENCIES>) => async (
-  dispatch: ReduxDispatch<GlobalState>,
-) => {
+const fetchCurrencySelectedValue = (
```

Comment:
In english language adjective goes in front of a noun:

https://learnenglish.britishcouncil.org/english-grammar-reference/adjective-order

so `fetchCurrencySelectedValue` should become `fetchSelectedCurrencyValue` or even `fetchCurrencyValue`

OR

as I can see that this action calls `setCurrencyRate` later, why can't it even be `fetchCurrencyRate` then? (I think it's more correct)

Reply:
Fixed